_posts/2015-08-06-Rust-1.2.md

```diff
@@ -0,0 +1,272 @@
+---
+layout: post
+title: "Announcing Rust 1.2"
+author: The Rust Core Team
+---
+
+Today marks the end of another Rust release cycle: we are [releasing][install]
```

Comment:
This sentence seems a bit awkward/doesn't suck me in (e.g. saying "release" twice, and the "end" seems negative).